core_lib/src/structure/object.cpp

```diff
@@ -61,6 +61,8 @@ void Object::init()
 
     // default palette
     loadDefaultPalette();
+
+    createDefaultLayers();
```

Comment:
`createDefaultLayers()` This method is intentionally not placed inside the `Object::init()`. Doing so breaks many unit tests.